### PR TITLE
improve list page's context

### PR DESCRIPTION
**Pull request by itzCozi**
improve the list page context with a tooltip and header

### CODE REVIEW

**Comment by itzCozi** [comment]
got another pr for the `theme-settings` branch once we merge

**Comment by itzCozi** [comment]
also mb for formatting issues

**Comment by itzCozi** [comment]
image

File: src/components/layout/sheet-menu.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function SheetMenu() {
         <SheetHeader>
           <Button className="mt-3 flex items-center justify-center" variant="link" asChild>
             <Link to="/" className="flex items-center gap-2" search={{ q: "" }} onClick={() => setIsOpen(false)}>
-              {theme === "dark" ? <img src={LogoHeader} className="h-12" /> : <img src={LogoHeaderDark} className="h-12" />}
+              {theme === "dark" ? <img src={LogoHeader} className="h-13" /> : <img src={LogoHeaderDark} className="h-13" />}
```
Comment (JorrinKievit) [inline src/components/layout/sheet-menu.tsx:27]: h-13 is not a valid tailwind class

File: src/routes/lists.tsx [hunk 2]
```diff
@@ -12,7 +13,22 @@ export function Lists() {
   return (
     <div className="flex flex-1 justify-center">
       <div className="flex w-full flex-col gap-4">
-        {bookmarks.length > 0 ? <h1 className="text-2xl font-bold">Your Bookmarks</h1> : null}
+        {bookmarks.length > 0 ? (
+          <h1 className="text-2xl font-bold">Your Bookmarks</h1>
+        ) : (
+          <div>
+            <TooltipProvider delayDuration={0}>
```
Comment (JorrinKievit) [inline src/routes/lists.tsx:20]: Don't think we should use tooltips (for mobile users that can't hover)

**Comment by itzCozi** [comment]
fixed

File: src/routes/lists.tsx [hunk 3]
```diff
@@ -12,7 +12,14 @@ export function Lists() {
   return (
     <div className="flex flex-1 justify-center">
       <div className="flex w-full flex-col gap-4">
-        {bookmarks.length > 0 ? <h1 className="text-2xl font-bold">Your Bookmarks</h1> : null}
+        {bookmarks.length > 0 ? (
+          <h1 className="text-2xl font-bold">Your Bookmarks</h1>
+        ) : (
+          <div>
+            <h1 className="text-2xl font-bold">No Bookmarks</h1>
+            <p className="text-sm">Add some books here by clicking the bookmark icon on a book card.</p>
```
Comment (JorrinKievit) [inline src/routes/lists.tsx:20]: Maybe more along the lines of: Start adding some books using the bookmark button. Start searching here <add link to search page> ?

**Review by JorrinKievit — APPROVED** [review]

**Pull request merged**